### DISCUSSION TITLE
Sharing SSH keys

### DISCUSSION THREAD

**Discussion by leocavalcante** [devcontainers/community#124] [Q&A]
It this ok?
https://github.com/opencodeco/phpctl/blob/main/.devcontainer/devcontainer.json#L9-L12
Doesn't feel right to me.
It this the way to share SSH keys inside the container so I can use `git` and VSCode's Source Control?
Or there is a proper way to do that?

**Comment by samruddhikhandale** [comment]
Hi 👋 

The Feature would not help to share the SSH keys, however, you'd need to mount them. See one such example of `.devcontainer.json`

```jsonc
{
  ....
 "mounts": ["source=/path/to/.ssh,target=~/.ssh,type=bind,consistency=cached"]
}

```

**Reply by leocavalcante** [reply]
Sorry, it was override by other commits, but the point in file when a I created the question was: https://github.com/opencodeco/phpctl/blob/0ce2ee2e54b046e3fd6c370e2a4ab2e976a47009/.devcontainer/devcontainer.json#L10
